Dashboard/app/js/lib/views/surveys/question-view.js

```diff
@@ -115,7 +115,7 @@ FLOW.QuestionView = FLOW.View.extend({
 
   amNoOptionsType: function () {
     var val;
-    if (!Ember.none(this.type)) {
+    if (!Ember.none(this.type)) { 
```

Comment:
Please remove the whitespace at the end of the line. 👍

Reply:
iam not sure about the end of the line you are referring too.